### ISSUE TITLE
Why is the number of SNPs in a cell type the same to the control's one?

### ISSUE THREAD

**Issue by hirokoohmiya** [bulik/ldsc#393]
Hello,
I am trying the cell-type specific S-LDSC calculation by following the tutorial using Cahoy dataset. According to the manual, *.ldcts file should be a set of LD scores corresponding to the specifically expressed genes in the cell type and the LD scores of all genes as a control. I think that the number of expressed genes in one cell type should be different from all genes contained in control LD score set. However, ldsc.py says that the number of SNPs included in the LD score file must be the same between a cell-type and control. Could you tell me the way how to calculate LD scores for specifically expressed genes in the cell type with the same SNPs to the control LD score?

Tutorial:
https://github.com/bulik/ldsc/wiki/Cell-type-specific-analyses

**Comment by aksarkar** [comment]
You are correct that the specifically expressed genes (SEG) in one cell type are in general different from the genes in the control set.

However, every SNP (regardless of whether it is near an SEG) has an LD score.

Therefore, you compute an LD score for every SNP even when considering SEG in a specific cell type.

You can do this as described in https://github.com/bulik/ldsc/wiki/LD-Score-Estimation-Tutorial#partitioned-ld-scores

**Comment by 809825706** [comment]
> You are correct that the specifically expressed genes (SEG) in one cell type are in general different from the genes in the control set.
> 
> However, every SNP (regardless of whether it is near an SEG) has an LD score.
> 
> Therefore, you compute an LD score for every SNP even when considering SEG in a specific cell type.
> 
> You can do this as described in https://github.com/bulik/ldsc/wiki/LD-Score-Estimation-Tutorial#partitioned-ld-scores

If I use the bed file from ATACseq to make cell type annotation, how should I compute an LD score as control?
